tutorials/workflow/csharp/combined-patterns/WorkflowApp/Program.cs

```diff
@@ -0,0 +1,87 @@
+using Dapr.Client;
+using Dapr.Workflow;
+using WorkflowApp;
+using WorkflowApp.Activities;
+
+var builder = WebApplication.CreateBuilder(args);
+builder.Services.AddSingleton<HttpClient>(DaprClient.CreateInvokeHttpClient(appId: "shipping"));
```

Comment:
While this works in the short term, it shouldn't be best guidance going forward.